packages/react/src/environments.ts

```diff
@@ -66,3 +74,20 @@ export const staging: EnvironmentConfig = {
   },
   handleResolver: (handle) => `${handle}.test`,
 };
+
+/**
+ * The development environment configuration
+ *
+ * @deprecated Please use the {@link development} variable instead
+ *
+ * After extensive considerations, we have decided to rename the `staging` variable into `development`.
+ * We found that the term `staging` is inflated and not very clear to the all users.
+ *
+ * Together with {@link `production`} the changes is meant to be more explicit about the intended usage of these variables.
+ *
+ * - `production` is the environment to be used in the live instance of your application (real users, real profiles, real data).
+ * - `development` is the environment to be used when you develop and test your application (test users, test profiles, test data).
+ *
+ * We also aligned the naming in the `@lens-protocol/client` package to enable interoperability between the two packages.
+ */
```

Comment:
```suggestion

/**
 * The development environment configuration
 *
 * @deprecated Please use the {@link development} variable instead
 *
 * After extensive considerations, we have decided to rename the `staging` variable into `development`.
 * We found that the term `staging` is inflated and unclear to all users.
 *
 * Together with {@link `production`} the changes is meant to be more explicit about the intended usage of these variables.
 *
 * - `production` is the environment to be used in the live instance of your application (real users, real profiles, real data).
 * - `development` is the environment to be used when you develop and test your application (test users, test profiles, test data).
 *
 * We also aligned the naming in the `@lens-protocol/client` package to enable interoperability between the two packages.
 */
```

Reply:
I see, I'll reword it "unclear to all users." is not what I was aiming to say. Thank for flagging it as not clear.